apps/happy-blocks/src/pricing-plans/edit.scss

```diff
@@ -1 +1,165 @@
-// placeholder for edit styles
+@import "~@automattic/typography/styles/variables";
+@import "~@automattic/color-studio/dist/color-variables";
+
+
+/*
+Note:
+  There are several usages of !important in this file due to 104-gh-Automattic/lighthouse-forums.
+	Once this issue is resolved, we can remove the !important declarations.
+*/
+
+@keyframes wp-block-a8c-pricing-plans__skeleton__animation {
+	0% {
+		opacity: 0.5;
+	}
+
+	50% {
+		opacity: 1;
+	}
+
+	100% {
+		opacity: 0.5;
+	}
+}
+
+.wp-block-a8c-pricing-plans {
+	display: flex;
+	padding: 28px;
+	margin: 24px 0;
+	border: 1px solid $studio-gray-5;
+	border-radius: 2px;
+	gap: 32px;
+	background-color: $studio-white;
+
+	&__settings-plan select {
+		min-height: 30px !important;
+		height: 30px !important;
+		padding: 0 26px 0 8px !important;
+		margin: 0 !important;
+	}
+
+	&__skeleton {
+		color: #eee;
+		background-color: #eee;
+		width: 100%;
+		margin: 24px 0;
+		min-height: 215px;
+
+		animation: wp-block-a8c-pricing-plans__skeleton__animation 1.6s ease-in-out infinite;
+
+		&::after {
+			content: "";
+		}
+	}
+
+	&__header {
+		flex: 1;
+
+		&-label {
+			font-family: $brand-serif;
+			font-weight: 400;
+			font-size: 2rem;
+			line-height: 1;
+			display: flex;
+			letter-spacing: -0.32px;
+			color: $studio-gray-100;
+		}
+
+		&-domain {
+			font-family: "SF Pro Text", $sans;
```

Comment:
Could we please put this into a constant and use that like in the other places in this file?